app/mailers/ineligible_participant_mailer.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+class IneligibleParticipantMailer < ApplicationMailer
+  ECT_PREVIOUS_INDUCTION_TEMPLATE = "e27eabc4-2d54-4356-8073-85cf9a559ce4"
+  MENTOR_PREVIOUS_PARTICIPATION_TEMPLATE = "d64d1f3b-d156-444c-bf05-b5254a0dd0ad"
+
+  ACTIVE_FLAGS_TEMPLATES = {
+    ect: "dcab5e33-c7c3-4d5a-84b6-458ae7640061",
+    mentor: "f2b63f96-241f-480b-8825-299d5576ae59",
+  }.freeze
+
+  NO_QTS_TEMPLATES = {
+    ect: "fbc5b291-68fd-42b6-bb9b-e09cac6affe5",
+    mentor: "08f36052-4073-47c9-9dc7-ddf79ab5b371",
+  }.freeze
```

Comment:
Should the top two be a hash like the bottom four?

Reply:
They are similarly named but for different things so thought it might be more obvious if separate